lib/platform-stats.js

```diff
@@ -18,23 +18,29 @@ export const updatePlatformStats = async (pgClient, honestMeasurements) => {
  * @param {import('./preprocess.js').Measurement[]} honestMeasurements
  */
 export const updateDailyStationStats = async (pgClient, honestMeasurements) => {
-  // TODO: when we add more fields, we will update the ON CONFLICT clause
-  // to update those fields, and we won't just use a Set for the stationIds
-  // which currently removes all granular measurement details
-  const stationIds = [
-    ...new Set(
-      honestMeasurements
-        .filter(m => m.stationId !== null)
-        .map(m => m.stationId)
+  const measurementCountsPerStation = new Map()
+  for (const m of honestMeasurements) {
+    if (m.stationId == null) continue
+
+    measurementCountsPerStation.set(
+      m.stationId,
+      (measurementCountsPerStation.get(m.stationId) || 0) + 1
     )
-  ]
-  debug('Updating daily station stats, unique_count=%s', stationIds.length)
+  }
+  debug('Updating daily station stats, station_count=%s', measurementCountsPerStation.size)
 
+  // Use EXCLUDED to allow incrementing the count
```

Comment:
```suggestion
```

This is kind of how these queries work, it's a common approach. I don't think it warrants a comment.